backend/Match/views.py

```diff
@@ -39,7 +40,7 @@ def get(self, request):
 class LaunchTestMatchView(APIView):
 	@method_decorator(must_be_authenticated)
 	def get(self, request):
-		token = MatchToken.objects.create_single_match_token(request.user, request.user)
+		token = MatchTokenManager.create_single_match_token(request.user, request.user)
 		match = MatchSetupManager.create_match_and_its_players(token)
 		if not match:
 			return Response({'error': 'Something went wrong when creating the match and players'}, status=status.HTTP_403_FORBIDDEN)
```

Comment:
the forbidden here is still wrong. I guess the 500 or Bad Gateway is best, if this is a db issue..... but not the best idea as previously discussed

Reply:
Let's fix this in issue #195